### PR TITLE
[lyra2-file-encryptor] Add new package port

### PR DESCRIPTION
**Pull request by 90felipe09**
# Add new package port: [Lyra2FileEncryptor]

Lyra2FileEncryptor is a C++ package with AES file encryption and Lyra2 password hash scheme. Lyra2 is one of the newest candidate upgrade in password hash scheme in PHS competition.

This package provides:

- File encryption feature;
- File decryption feature;
- Get decryption content from an encrypted file feature;
- Lyra2 method;

### CODE REVIEW

**Comment by JackBoosY** [comment]
Hi @90felipe09, thanks for this PR!
Please sign CLA first.

**Comment by 90felipe09** [comment]
Signed!

**Review by Hoikas — COMMENTED** [review]
Why is the port not named lyra2-file-encryptor like the repository?

**Comment by JackBoosY** [comment]
@Hoikas Good question! And I think we should make the port name shortly as soon as possible.
Please download [this patch](https://dev.azure.com/vcpkg/public/_build/results?buildId=53251&view=artifacts&pathAsName=false&type=publishedArtifacts), apply and commit the changes.

**Comment by 90felipe09** [comment]
Thanks for the feedback. I will make these changes:
- change port name to `lyra2-file-encryptor`
- Apply the patch

File: versions/baseline.json [hunk 1]
```diff
@@ -3780,6 +3780,10 @@
       "baseline": "1.18.0",
       "port-version": 0
     },
+    "lyra2fileencryptor": {
+      "baseline": "1.2.0",
+      "port-version": 0
+    },
```
Comment (JackBoosY) [inline versions/baseline.json:3786]: Please remove these code and delete _versions/l-/lyra2fileencryptor.json_, rerun the `x-add-version` command then commit new changes.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Comment by JackBoosY** [comment]
Ping @90felipe09 for response.

**Comment by 90felipe09** [comment]
Sorry for the long standby period. I think that I will close this PR and figure out the Shared/Dynamic linking library compilation. I'm having some issues here with it. When accomplish it I will reopen it.

Thanks for all the support and feedback.

**Pull request closed without merging**